src/main/java/org/opentripplanner/updater/vehicle_positions/GtfsRealtimeFileVehiclePositionSource.java

```diff
@@ -0,0 +1,57 @@
+package org.opentripplanner.updater.vehicle_positions;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.google.transit.realtime.GtfsRealtime.FeedEntity;
+import com.google.transit.realtime.GtfsRealtime.FeedMessage;
+import com.google.transit.realtime.GtfsRealtime.VehiclePosition;
+import org.opentripplanner.routing.graph.Graph;
+import org.opentripplanner.updater.JsonConfigurable;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.InputStream;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * Responsible for importing file of GTFS-RT into memory
```

Comment:
```suggestion
 * Responsible for reading GTFS-rt vehicle positions from a local file and loading into memory.
```

Reply:
Added in 94fe2b070ea0ab38e10e88f37b271586688d48bd